admiral/envs/corridor/corridor.py

```diff
@@ -2,8 +2,7 @@
 from gym.spaces import Discrete
```

Comment:
Just remove the single agent version because it is a special case of the multiagent version.